tests/e2e/tidbcluster/tidbcluster.go

```diff
@@ -881,19 +903,21 @@ var _ = ginkgo.Describe("TiDBCluster", func() {
 	ginkgo.Context("[Feature: AutoFailover]", func() {
 		// TODO: explain purpose of this case
 		ginkgo.It("should clear TiDB failureMembers when scale TiDB to zero", func() {
-			ginkgo.By("Deploy initial tc with bad tidb pre-start script")
-			tcCfg := newTidbClusterConfig(e2econfig.TestConfig, ns, "tidb-scale", "admin", utilimage.TiDBV4)
-			tcCfg.Resources["pd.replicas"] = "3"
-			tcCfg.Resources["tikv.replicas"] = "1"
-			tcCfg.Resources["tidb.replicas"] = "1"
-
-			tcCfg.TiDBPreStartScript = strconv.Quote("exit 1")
-			oa.DeployTidbClusterOrDie(&tcCfg)
+			ginkgo.By("Deploy initial tc")
+			tc := fixture.GetTidbCluster(ns, "tidb-scale", utilimage.TiDBV4)
+			tc.Spec.PD.Replicas = 1
+			tc.Spec.TiKV.Replicas = 1
+			tc.Spec.TiDB.Replicas = 1
+			tc.Spec.TiDB.Config.Set("log.file.max-size", "300")
```

Comment:
What's this for?

Reply:
Use wrong TiDB config to create failureMembers.